### DISCUSSION TITLE
disable auto scrolling?

### DISCUSSION THREAD

**Discussion by programmer131** [emqx/MQTTX#657] [Q&A] [answered]
is there a way to disable auto scrolling? sometimes need to scroll up to view older messages but it auto scroll to bottom if new message arrive.

**Comment by oceanlvr** [comment]
Great device.

**Reply by oceanlvr** [reply]
We'll add it in the later version.

**Comment by ysfscream** [comment, marked as answer]
Now we support on https://github.com/emqx/MQTTX/releases/tag/v1.7.0
